upgrade.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -eux
+
+fetch_latest() {
+  # Fetch latest version from Github releases API
+  LATEST=$(curl -s "https://api.github.com/repos/moby/moby/releases" | jq -r '.[0].name')
```

Comment:
`.[0].name` is the release name. The rest of the logic expects the tag name.


```suggestion
  LATEST=$(curl -s "https://api.github.com/repos/moby/moby/releases" | jq -r '.[0].tag_name')
```

Reply:
Also, since it needs just one release, limit the requested numbers.

Reply:
Done here: https://github.com/canonical/docker-snap/pull/199/commits/c54813ebd3de10f9dfcf8a65f1fec05d85279564